### PR TITLE
Git Workshop 2023 Assignment!

### PR DESCRIPTION
**Pull request by aadarKdk**

### CODE REVIEW

File: 2023/assignments/aadarsha/aadarsha.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>Aadarsha Khadka</p>
+
+<p align="center">
+    <a href="#" target="blank"
```
Comment (suravshrestha) [inline 2023/assignments/aadarsha/aadarsha.md]: @aadrkdk, if you have social media links, kindly replace them. Otherwise, remove any existing links.

**Review by suravshrestha — CHANGES_REQUESTED** [review]

File: 2023/assignments/aadarsha/aadarsha.md (hunk 1 above)
Comment (aadarKdk) [reply 2023/assignments/aadarsha/aadarsha.md]: @suravshrestha, I will do it.

**Comment by aadarKdk** [comment]
I have updated my assignments.

**Comment by suravshrestha** [comment]
@aadrkdk, thank you for your active participation in the workshop. Your contributions are highly appreciated.

**Pull request merged**